_sass/index.scss

```
@@ -141,6 +141,7 @@
   .section-1-lights {
     max-width: 100%;
     padding: 0 16px;
+	margin-bottom: -15%;
```

Can you replace the tab by spaces? 
```suggestion
    margin-bottom: -15%;
```